docs/CLI.md

```diff
@@ -217,6 +217,12 @@ Allows the test suite to pass when no files are found.
 
 Run tests from one or more projects.
 
+### `--reporters`
+
+Run tests with specified reporters. [Reporter options](http://jestjs.io/docs/en/configuration#reporters-array-modulename-modulename-options) are not available via CLI. Example with multiple reporters:
```

Comment:
I think this link should be relative? So it's correct for versioned docs as well

Reply:
Good call - fixing now